cmd/kubeops/main.go

```diff
@@ -59,6 +60,11 @@ func main() {
 	withAgentConfig := handler.WithAgentConfig(storageForDriver, options)
 	r := mux.NewRouter()
 
+	// Healthcheck
+	health := healthcheck.NewHandler()
+	r.Handle("/live", health)
+	r.Handle("/ready", health)
```

Comment:
It appears tiller-proxy isn't a great example here... can you add a TODO to add app specific health and readiness checks as per https://github.com/heptiolabs/healthcheck ? Thanks.